public/plugin.tsx

```diff
@@ -99,6 +101,9 @@ export class AssistantPlugin
       return account;
     };
 
+    // setup ui trigger
+    bootstrap(setupDeps.uiActions);
```

Comment:
Seems `uiActions` is missing in the `opensearch_dashboard.json` file.

Reply:
Thanks, missed to commit that file, fixed now